src/test/java/code/practice/tasks/StreamTasksTest.java

```diff
@@ -0,0 +1,48 @@
+package code.practice.tasks;
+
+import code.practice.exceptions.UnexpectedMajorException;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class StreamTasksTest {
+    StreamTasks streamTasks = new StreamTasks();
+
+    @Test
+    void sumOfPositiveIntegersTest() {
+        var numbers = List.of("1", "-4", "5", "-1", "6");
+        assertEquals(12, streamTasks.sumOfPositiveIntegers(numbers));
+    }
+
+    @Test
+    void sumOfIntegersTest() {
+        var numbersLists = List.of(List.of("1", "-4", "5", "-1", "6"));
+        assertEquals(12, streamTasks.sumOfIntegers(numbersLists));
+    }
+
+    @Test
+    void justPrintSequenceTest() {
+        streamTasks.justPrintSequence(2, 9);
+    }
+
+    @Test
+    void getAnyNegativeIntAsStringTest() throws UnexpectedMajorException {
```

Comment:
After you fix your original method's code, please also add test to check exception.